src/containers/MyProfile/MyProfile.tsx

```diff
@@ -0,0 +1,206 @@
+import { Formik } from "formik";
+import React, { useCallback, useEffect, useMemo } from "react";
+import { Image, RefreshControl } from "react-native";
+import { styles } from "./MyProfile.styles";
+import { MyProfileFields, formHelper, validationSchema } from "./MyProfile.helper";
+import { Input, ApiCallAnimatedButton, Loader, Button } from "../../components";
+import { images } from "../../assets/images";
+import { ScrollView, TouchableOpacity } from "react-native-gesture-handler";
+import { useAuthContext, useRequestStatusContext } from "../../context";
+import { URLNames } from "../../constants";
+import { Optionable, RequestStatus } from "../../types";
+import { MyProfileStackProps } from "../../types/routes";
+
+const ADDRESS_SEPARATOR = "*";
+
+export const MyProfile = ({ navigation }: MyProfileStackProps<"myProfile/profile">) => {
+  const { requestStatuses, dispatch } = useRequestStatusContext();
```

Comment:
1. Weird behaviour, screen isn't scrollable
2. By tapping on the logout confirmation isn't appeared
<img width="634" alt="image" src="https://user-images.githubusercontent.com/36453308/213785950-2b5575dc-4f48-4426-9849-a05a54c7953d.png">
 see 2.4

Reply:
fixed

Reply:
by tapping onLogout wrong modal appeared
<img width="535" alt="image" src="https://user-images.githubusercontent.com/36453308/216572954-c08a187b-a8e1-43bb-af08-23d2fe938c1c.png">